packages/api-security/src/scopes.ts

```diff
@@ -1,31 +1,28 @@
-import { rule } from "graphql-shield";
+import { GraphQLFieldResolver } from "@webiny/graphql/types";
 
-export const hasScope = (scope: string) => {
-    return rule()(async (parent, args, ctx) => {
-        if (!ctx.user) {
-            return false;
-        }
-
-        const access = await ctx.user.access;
-        if (access.fullAccess) {
-            return true;
-        }
+export class ScopeError extends Error {
```

Comment:
Let's call this `SecurityError`, calling it `ScopeError` is a bit vague.

Reply:
Please include `code` and `data` properties in the error. Like we have for example in `packages/graphql/src/responses.ts:16`.

This way, in GQL, you'll be able to get the error code.

![image](https://user-images.githubusercontent.com/5121148/80338932-e5629280-885d-11ea-9973-5d31e6872d95.png)